README.md

```diff
@@ -85,6 +85,23 @@ Installation
 8. Set up your webserver. The Configuration section below describes how
    to setup the rewrite rules for both nginx and apache.
 
+Installation with Docker
+========================
+
+This setup uses [docker-compose] to orchestrate docker containers.
+
+1. Clone or download `xhgui` from GitHub.
+
+2. Startup the containers: `docker-compose up -d`
+
+3. Open your browser at http://xhgui.127.0.0.1.xip.io:27017 or just http://localhost:27017
```

Comment:
doesn't this point the browser to the MongoDB instance? What woudl be the point of that?

Reply:
too soon! pushed update!